numba/np/ufunc/parallel.py

```diff
@@ -344,9 +344,9 @@ def _check_tbb_version_compatible():
     try:
         # first check that the TBB version is new enough
         if _IS_WINDOWS:
-            libtbb_name = 'tbb'
+            libtbb_name = 'tbb12.dll'
```

Comment:
What happens if there is an older TBB version? Was there a previous version of TBB where the library was named `tbb.dll` that would be missed now?

Reply:
... or was this just always wrong?

Reply:
Am trying to enforce some consistency in https://github.com/numba/numba/pull/7606, whereby the CI system checks that if a threading layer is requested that it can actually be loaded.